src/cascadia/TerminalApp/PowershellCoreProfileGenerator.cpp

```diff
@@ -135,7 +135,8 @@ namespace
     };
 }
 
-using namespace ::TerminalApp;
+using namespace TerminalApp;

```

Comment:
why did you drop the `::` here? just curious..

Reply:
I...don't know why I did that haha